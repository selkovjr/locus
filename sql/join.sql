--
--  Locus datatype test
--
SELECT count(*) FROM test_locus t1 JOIN test_locus t2 ON t1.p = t2.p;
SELECT count(*) FROM test_locus t1 JOIN test_locus t2 ON t1.p && t2.p;
