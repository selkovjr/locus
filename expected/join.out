--
--  Locus datatype test
--
SELECT count(*) FROM test_locus t1 JOIN test_locus t2 ON t1.p = t2.p;
  count  
---------
 7198818
(1 row)

SELECT count(*) FROM test_locus t1 JOIN test_locus t2 ON t1.p && t2.p;
  count   
----------
 10655486
(1 row)

